src/Executor/ExecutionProcessFactory.php

```diff
@@ -45,8 +45,8 @@ public function __construct($consolePath, $processTimeout, $environment)
      */
     public function create($uuid)
     {
-        return $process = (new Process(
+        return $process = (new Process([
```

Comment:
I don't think this works correctly as you can not use longer `spaces` without fromShellCommandline:

```
if (method_exists(Process::class, 'fromShellCommandline') {
    $process = Process::fromShellCommandline(implode(' ', [$this->consolePath, 'task:execute', $uuid, '--env=' . $this->environment]));
}

$process = // old way

$process->setTimeout($this->processTimeout);
```

Also not sure if it would make here more sense using the `PhpProcess` class instead of `Process`